Assets/Scripts/LevelManager.cs

```diff
@@ -1,18 +1,33 @@
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
 
 public class LevelManager : MonoBehaviour
 {
     private int totalPoints = 0;
     private int currentPoints = 0;
 
-    // Start is called before the first frame update
+    public Button[] levelsButtons;

```

Comment:
LevelManager is actually a script intended to be used only in levels. For the menu you should make another script, like LevelsMenu. Also I think you can merge SceneSwitcher into LevelsMenu as well, there's no point for having two separate scripts.
Please remove all these changes from LevelManager.

Reply:
Done